games-emulation/bsnes_libretro/bsnes_libretro-0.94.git20171227.recipe

```diff
@@ -0,0 +1,64 @@
+SUMMARY="A port of bsnes to the libretro architecture"
+DESCRIPTION="bsnes is an accurate SNES emulator, but can also emulate the NES, \
+Game Boy (DMG), Game Boy Color, and Game Boy Advance systems. This package \
+provides three different profiles: one focuses on accuracy but can be \
+CPU-intensive, one focuses on speed but is less accurate, and one is \
+attempting to provide a reasonable balance between the previous two."
+HOMEPAGE="https://byuu.org/emulation/higan/"
+COPYRIGHT="2014-2018 byuu, the libretro team"
+LICENSE="GNU GPL v3"
+REVISION="1"
+srcGitRev="4e572a3cfa10e56bedb6737b0494e8165f6490d5"
+SOURCE_URI="https://github.com/libretro/bsnes-libretro/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="77255571b654ef1b7559608818393dc8377c657dc30aa7d3ec45e40662c99a31"
+SOURCE_DIR="bsnes-libretro-$srcGitRev"
+ADDITIONAL_FILES="
+	bsnes_accuracy_libretro.info.in
+	bsnes_balanced_libretro.info.in
+	bsnes_performance_libretro.info.in
+	"
+PATCHES="bsnes_libretro-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	bsnes_libretro$secondaryArchSuffix = $portVersion
+	lib:bsnes_accuracy_libretro$secondaryArchSuffix = $portVersion
+	lib:bsnes_balanced_libretro$secondaryArchSuffix = $portVersion
+	lib:bsnes_performance_libretro$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+SUPPLEMENTS="
+	retroarch$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:make
+	cmd:gcc$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	for bsnesProfile in "accuracy" "balanced" "performance"; do
+		sed -e "s/@DISPLAY_VERSION@/v${portVersion%~.*}/" \
```

Comment:
```diff
-		sed -e "s/@DISPLAY_VERSION@/v${portVersion%~.*}/" \
+		sed -e "s/@DISPLAY_VERSION@/v${portVersion%.git*}/" \
```
This is in case you don't want the `.git20171227` to appear in the core info.
BTW, we forgot to make a similar change in fbalpha_libretro after we renamed the recipe. But that's OK. We can fix it at any time.